server/grpc.go

```diff
@@ -1,17 +1,52 @@
 package server
 
 import (
+	"time"
+
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
+	"google.golang.org/grpc"
 
+	"github.com/openfresh/plasma/config"
 	"github.com/openfresh/plasma/event"
+	"github.com/openfresh/plasma/log"
 	"github.com/openfresh/plasma/manager"
 	"github.com/openfresh/plasma/metrics"
 	"github.com/openfresh/plasma/protobuf"
 	"github.com/openfresh/plasma/pubsub"
 	"github.com/pkg/errors"
 )
 
+type GRPCServer struct {
```

Comment:
__[golint]__
* [golint] exported type GRPCServer should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->